packages/caliper-fabric/lib/identity-management/IdentityManager.js

```diff
@@ -191,7 +191,14 @@ class IdentityManager {
      * @private
      */
     async _extractIdentitiesFromWallet(mspId, wallet) {
-        // TODO: To be implemented
+        const walletFacade = await this.walletFacadeFactory.create(wallet.path);
+        const allIDNames = await walletFacade.getAllIdentityNames();
```

Comment:
could you change `allIDNames` to `allIdentityNames` as we don't want to use abbreviations